Makefile

```diff
@@ -111,7 +111,7 @@ v8:
 	tools/make-v8.sh v8
 	$(MAKE) -C deps/v8 $(V8_ARCH).$(BUILDTYPE_LOWER) $(V8_BUILD_OPTIONS)
 
-test: | cctest  # Depends on 'all'.
+test: | build-addons cctest  # Depends on 'all'.
```

Comment:
Ah sorry about that. I was comparing this with the Makefile in master and incorrectly added this to the `test` target. It looks like the correct way to run the addons test is using `make test-addons`. I'll remove `build-addons` from the `test` target and check that this works with the `test-addons` target. Thanks!